understanding/20/keyboard.html

```diff
@@ -69,6 +69,32 @@ <h2>Intent of Keyboard</h2>
          
       </p>
       
+      <div class="note">
+         <p>Platforms and user agents usually have conventions for how web content or
+            applications are controlled with a keyboard interface. If content does not follow
+            the platform/user agent conventions it may be difficult to use, as users will need
+            to learn different interaction methods. As a <em>best practice</em>, content
+            should follow the platform/user agent conventions. However, deviating from these
+            conventions does <em>not</em> fail the normative requirement of this Success Criterion.</p>
+         <p>
+            For instance, buttons that have focus can generally be activated using both the
+            <kbd>Enter</kbd> key and the <kbd>Space</kbd> bar. If a custom button control
+            in a web application instead only reacts to <kbd>Enter</kbd>
+            (or even a completely custom key or key combination), this still
+            <strong>passes</strong> the requirements of this Success Criterion.
```

Comment:
```suggestion
            <strong>satisfied</strong> the requirements of this Success Criterion.
```

Reply:
the correct replacement here would then be "satisfies" (rather than "satisfied")

Reply:
resolved in https://github.com/w3c/wcag/pull/1642/commits/38cce8aada87e526bc2fe0e1449b073828672819